source/_components/knx.markdown

```diff
@@ -13,23 +13,95 @@ ha_release: 0.24
 ha_iot_class: "Local Polling"
 ---
 
-[KNX/EIB](http://www.knx.org) integration for Home Assistant allows you to connect to a KNX bus. The component requires a local KNX/IP interface like the [Weinzierl 730](http://www.weinzierl.de/index.php/en/all-knx/knx-devices-en/knx-ip-interface-730-en). Through this it will send and receive commands to and from other devices to the KNX bus.
+Overview
+--------
 
-There is currently support for the following device types within Home Assistant:
+The [KNX](http://www.knx.org) integration for Home Assistant allows you to connect to a KNX/IP devices.
+
+The component requires a local KNX/IP interface like the [Weinzierl 730](http://www.weinzierl.de/index.php/en/all-knx/knx-devices-en/knx-ip-interface-730-en). Through this it will send and receive commands to and from other devices to the KNX bus.
 
+There is currently support for the following device types within Home Assistant:
+ 
 - [Binary Sensor](/components/binary_sensor.knx)
 - [Cover](/components/cover.knx)
 - [Sensor](/components/sensor.knx)
 - [Switch](/components/switch.knx)
 - [Light](/components/light.knx)
 - [Thermostat](/components/climate.knx)
+- [Notify](/components/notify.knx)
+
+Configuration
+--------------
+
+To use your KNX in your installation, add the following lines to your `configuration.yaml` file:
 
-A `knx` section must be present in the `configuration.yaml` file and contain the following options as required:
+```yaml
+knx:
+```
+
+Optional, recommended for large KNX installations (>100 devices) and/or if you want to use the XKNX abstraction also for other scripted tools outside HASS:
 
 ```yaml
-# Example configuration.yaml entry
 knx:
+  config_file: '/path/to/xknx.yaml'
 ```
+* **config_file**: (*Optional*) path for xknx configuration file.
+
+If the autodetection of the KNX/IP device does not work you can specify ip/port of the tunneling device:
+
+```yaml 
+knx:
+  tunneling:
+    host: '192.168.2.23'
+    port: 3671
+    local_ip: '192.168.2.109'
+```
+* **host**: Host of KNX/IP tunneling device
+* **port**: Port of KNX/IP tunneling device
+* **local_ip**: IP of local interface
```

Comment:
**Spelling & Grammar**:
- `IP of local interface` -> `IP of the local interface`